app/main.py

```diff
@@ -1,2 +1,168 @@
+from typing import Any, List
+
+from app.point import Point
+
+
 class Dictionary:
-    pass
+    __items: List[tuple | None]
+    __current_len: int = 0
+
+    def __init__(
+        self,
+        args: List[tuple[Any, Any] | list[Any, Any]] | None = None
```

Comment:
You should use either objects from `typing` or built-in data types for type hinting, do not mix them.